src/NuProj.Targets/NuProj.targets

```diff
@@ -984,150 +880,179 @@
+      <Output TaskParameter="DestinationFiles" ItemName="FileWrites"/>
 
-         Note: We should forward calling the Clear target when building from the command line.
-               Assuming, of course, we're not suppressing building project references. -->
+    </Copy>    
+  </Target>
 
-    <MSBuild Targets="Clean"
-             Condition="'$(BuildProjectReferences)' == 'true'"
-             Projects="@(ProjectReferenceWithConfiguration)"
-             Properties="%(ProjectReferenceWithConfiguration.SetConfiguration); %(ProjectReferenceWithConfiguration.SetPlatform)" />
+  <Target Name="_CheckSymbolsPackageProduced"
+          AfterTargets="_CheckForCompileOutputs">
 
-    <!-- Delete our outputs and intermediates. -->
+    <!-- Record the .symbols.nupkg if one was produced. -->
+    <PropertyGroup>
+      <_SymbolsPackageProduced Condition="!Exists('@(IntermediateSymbolsPackage)')">false</_SymbolsPackageProduced>
+    </PropertyGroup>
 
     <ItemGroup>
-      <_ToBeDeleted Include="$(NuSpecPath)" />
-      <_ToBeDeleted Include="$(NuGetOutputPath)" />
-      <_ToBeDeleted Include="$(NuGetSymbolsOutputPath)" />
+      <FileWrites Include="@(IntermediateSymbolsPackage)" Condition="'$(_SymbolsPackageProduced)'=='true'"/>
     </ItemGroup>
-    <Delete Files="@(_ToBeDeleted)" />
-
   </Target>
-  
+
   <!--
-    ============================================================
-                                        BeforeClean
+    ===================================================================================================================
+    ResolveAssemblyReferences
+    ===================================================================================================================
 
-    Redefine this target in your project in order to run tasks just before Clean.
-    ============================================================
-    -->
-  <Target Name="BeforeClean"/>
+    Dummy target to keep Microsoft.Common.targets target chain working.
 
-  <!--
-    ============================================================
-                                        AfterClean
+    
+    OUTPUTS:
+        @(ReferencePath) - Paths to resolved primary packages.
 
-    Redefine this target in your project in order to run tasks just after Clean.
-    ============================================================
-    -->
-  <Target Name="AfterClean"/>
-  
+    =================================================================================================================== -->
   <PropertyGroup>
-    <BuildDependsOn>
-      BeforeBuild;
-      CoreBuild;
-      AfterBuild
-    </BuildDependsOn>
+    <ResolveAssemblyReferencesDependsOn>
+      $(ResolveAssemblyReferencesDependsOn);
+      ResolvePackageReference
+    </ResolveAssemblyReferencesDependsOn>
   </PropertyGroup>
 
-  <Target Name="Build"
-          DependsOnTargets="$(BuildDependsOn)" 
-          Returns="@(_NuGetOutputFile)" />
+  <Target Name="ResolveAssemblyReferences"
```

Comment:
Original `ResolveAssemblyReferences` depends on single target framework. Since we can reference projects with targeting different frameworks, easiest option was to disable it.